win32/tests/reals_win32_ut/reals_win32_ut.c

```diff
@@ -62,6 +64,7 @@ TEST_FUNCTION(check_all_c_pal_reals)
     REGISTER_SYNC_GLOBAL_MOCK_HOOK();
     REGISTER_ARITHMETIC_GLOBAL_MOCK_HOOK();
     REGISTER_UUID_GLOBAL_MOCK_HOOK();
+    REGISTER_EXECUTION_ENGINE_GLOBAL_MOCK_HOOK();

```

Comment:
>REGISTER_EXECUTION_ENGINE_GLOBAL_MOCK_HOOK [](http://example.com/codeflow?start=4&length=42)

We need another test that validates that all of the reals are getting tested! /s

Reply:
;)